framework/src/testing/arbitrary.rs

```diff
@@ -0,0 +1,15 @@
+//! Implementations of `proptest.arbitrary.Arbitrary` trait for
+//! various types.
+
+use crate::packets::MacAddr;
+use proptest::arbitrary::{any, Arbitrary, StrategyFor};
+use proptest::strategy::{MapInto, Strategy};
+
+impl Arbitrary for MacAddr {
```

Comment:
do we exercise this anywhere? Also, probably worth comments on strategies and why we have them, esp as we add more going forward.

Reply:
yeah in the other project. decided not to doc these because the usage will be identical to any built in proptest supported types. there's no difference. `any::<MacAddr>()` gives you a strategy that generates a random `MacAddr`. the strategies not attached to a type will be documented on their usage if needs be.